---
title: Home
---

# Evolutionary Genomics of _Drosophila_

### We study the antagonistic coevolution between selfish genetic elements and their host genome as well as the evolution of 3D genome organization
{:.center}

{% include section.html full=true %}

{% include banner.html image="images/ATGC.png" %}

{% include section.html %}

# Highlights

{% capture text %}
#### Host-TE conflicts

#### Evolution of the 3D genome
The genomes of most metazoan species are organized into domains of interacting chromatin known as TADs, whose purpose is believed to be related to regulating gene expression by preventing improper enhancer/promoter contacts. We are using Hi-C chromosome conformation capture to study the evolutionary conservation and divergence of TADs and TAD boundaries across Drosophila. Contrary to early studies, we find that overall, the 3D genome evolves relatively rapidly. Furthermore, we do not find significant evidence for a relationship between TAD reorganization and differential gene expression between species. Importantly, we find that the rate of TAD evolution depends on chromatin state, with TADs enriched for developmentally regulated chromatin being the most conserved. Our results raise the possibility that there are different subtypes of TADs that evolve under different selective regimes. Reorganization of TADs enriched for housekeeping genes is likely to be tolerated more than TADs enriched for developmental genes, potentially due to the increased number of long-distance regulatory elements associated with genes involved in development.

{%
  include link.html
  link="research"
  text="See what we've published"
  icon="fas fa-arrow-right"
  flip=true
%}
{:.center}
{% endcapture %}

{%
  include feature.html
  image="images/DALL-E_fly.png"
  link="research"
  title="Our Research"
  text=text
%}

